core/store/migrations/0040_create_job_proposals.go

```diff
@@ -0,0 +1,36 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up40 = `
+CREATE TABLE job_proposals (
+    id BIGSERIAL PRIMARY KEY,
+	spec TEXT NOT NULL,
+	status VARCHAR (50) NOT NULL,
+	job_id uuid NOT NULL,
+	feeds_manager_id int NOT NULL,
+    created_at timestamp with time zone NOT NULL,
+    updated_at timestamp with time zone NOT NULL,
+	CONSTRAINT fk_feeds_manager FOREIGN KEY(feeds_manager_id) REFERENCES feeds_managers(id)
+);
+CREATE INDEX idx_job_proposals_job_id on job_proposals (job_id);
+CREATE INDEX idx_job_proposals_feeds_manager_id on job_proposals (feeds_manager_id);
```

Comment:
So there can be:

- 0 or more proposals per job
- 0 or more proposals per feeds manager

Is that correct?

Reply:
The job proposal will map 1:1 with a job. The `job_id` will be used as the `external_job_id` in a JPV2 job

There can be 0 or more proposals per feeds manager

Reply:
Then, the relationship should be reversed - job should have a nullable job_proposal_id and